tasks/sep/032-op-pectra-defence/VALIDATION.md

```diff
@@ -0,0 +1,126 @@
+# Validation
+
+This document can be used to validate the state diff resulting from the execution of the upgrade transaction.
+
+For each contract listed in the state diff, please verify that no contracts or state changes shown in the Tenderly diff
+are missing from this document. Additionally, please verify that for each contract:
+
+- The following state changes (and none others) are made to that contract. This validates that no unexpected state
+  changes occur.
+- All addresses (in section headers and storage values) match the provided name, using the Etherscan and Superchain
+  Registry links provided. This validates the bytecode deployed at the addresses contains the correct logic.
+- All key values match the semantic meaning provided, which can be validated using the storage layout links provided.
+
+## Nested Safe State Overrides and Changes
```

Comment:
As best practice, even for sepolia, we should make sure to include the section for the expected hashes at the top here, like this: https://github.com/ethereum-optimism/superchain-ops/blob/506bdc63f1c1143e3506532f2ed05a39dca515a0/tasks/eth/030-soneium-holocene-system-config-upgrade/VALIDATION.md#L12

screenshot of the above link:

<img width="1162" alt="image" src="https://github.com/user-attachments/assets/9eff0283-06e2-4d33-882f-8ec6e4ca3d19" />